src/baldr/graphreader.cc

```diff
@@ -203,39 +168,22 @@ TileCache* TileCacheFactory::createTileCache(const boost::property_tree::ptree&
 }
 
 // Constructor using separate tile files
-GraphReader::GraphReader(const boost::property_tree::ptree& pt)
-    : tile_hierarchy_(pt.get<std::string>("tile_dir")),
-      tile_extract_(get_extract_instance(pt)),
+GraphReader::GraphReader(const std::shared_ptr<GraphTileStorage>& tile_storage, const boost::property_tree::ptree& pt)
+    : tile_hierarchy_(tile_storage),
       cache_(TileCacheFactory::createTileCache(pt)) {
-  // Reserve cache (based on whether using individual tile files or shared,
-  // mmap'd file
-  cache_->Reserve(tile_extract_->tiles.empty() ? AVERAGE_TILE_SIZE : AVERAGE_MM_TILE_SIZE);
+  // Reserve cache
+  cache_->Reserve(AVERAGE_TILE_SIZE);
```

Comment:
sorry yes that is what i meant. it basically neuters the cache for tars by allowing it to have less tiles. sorry for the completely botched wording!

yes please if you could add that i think its probably the only blocker for merging this other than some basic testing we'd do on our end.